easybuild/easyconfigs/r/R/R-3.1.2-goolf-1.5.14.eb

```diff
@@ -0,0 +1,275 @@
+    ('R2OpenBUGS', '3.2-2.2', ext_options),
+    ('akima', '0.5-11', ext_options),
+    ('rjags', '3-14', ext_options),
+    ('R2jags', '0.04-03', ext_options),
+    ('DBI', '0.3.1', ext_options),
+    ('xtable', '1.7-4', ext_options),
+    ('RSQLite', '1.0.0', ext_options),
+    ('locfit', '1.5-9.1', ext_options),
+    ('lpSolve', '5.6.10', ext_options),
+    ('glmmML', '1.0', ext_options),
+    ('survey', '3.30-3', ext_options),
+    ('rlecuyer', '0.3-3', ext_options),
+    ('crs', '0.15-23', ext_options),
+    ('dclone', '2.0-0', ext_options),
+    
+    ('caTools', '1.17.1', ext_options),
+    ('gtools', '3.4.1', ext_options),
+    ('gdata', '2.13.3', ext_options),
+    ('gplots', '2.14.2', ext_options),
+    ('gsalib', '2.0', ext_options),
+    ('inline', '0.3.13', ext_options),
+    ('RcppArmadillo', '0.4.500.0', ext_options),
+    
+    ('XML', '3.98-1.1', ext_options),
+    ('spgrass6', '0.8-6', ext_options),
+
+    ('checkmate', '1.5.0', ext_options),
+    ('BBmisc', '1.8', ext_options),
+    ('fail', '1.2', ext_options),
+    ('brew', '1.0-6', ext_options),
+    ('base64enc', '0.1-2', ext_options), 
+    ('sendmailR', '1.2-1', ext_options),
+    ('BatchJobs', '1.5', ext_options),
+
+    ('BiocInstaller', '1.16.1', bioconductor_options),
+    ('BiocGenerics', '0.12.1', bioconductor_options),
+    ('Biobase', '2.26.0', bioconductor_options),
+    ('S4Vectors', '0.4.0', bioconductor_options),
+    ('IRanges', '2.0.0', bioconductor_options),
+    ('XVector', '0.6.0', bioconductor_options),
+    ('zlibbioc', '1.12.0', bioconductor_options),
+    ('Biostrings', '2.34.0', bioconductor_options),
+    ('GenomeInfoDb', '1.2.3', bioconductor_options),
+    ('AnnotationDbi', '1.28.1', bioconductor_options),
+    ('GenomicRanges', '1.18.1', bioconductor_options),
+    ('Rsamtools', '1.18.2', bioconductor_options),
+    ('BiocParallel', '1.0.0', bioconductor_options),
+    ('GenomicAlignments', '1.2.1', bioconductor_options),
+    ('rtracklayer', '1.26.2', bioconductor_options),
+    ('BSgenome', '1.34.0', bioconductor_options),
+    ('graph', '1.44.0', bioconductor_options),
+    ('annotate', '1.44.0', bioconductor_options),
+    ('genefilter', '1.48.1', bioconductor_options),
+    ('geneplotter', '1.44.0', bioconductor_options),
+    ('DESeq', '1.18.0', bioconductor_options),
+    ('DESeq2', '1.6.2', bioconductor_options),
+    ('ShortRead', '1.24.0', bioconductor_options),
+
+    ('rlecuyer', '0.3-3', ext_options),
+    ('pbdMPI', '0.2-5', ext_options),
+    ('pbdSLAP', '0.1-9', ext_options),
+    ('pbdBASE', '0.2-3', ext_options), # with icc: Rtools/tools.c(41): error: incorrect use of va_start
+    ('pbdDMAT', '0.2-3', ext_options), # depends on pdbBASE
+    #('pbdNCDF4', '0.1-4', ext_options),  # add parallel netCDF as a dependency if this gets added
+
+    #('BH', '1.54.0-5', ext_options), # Boost headers for rstan
+    #('rstan', '2.4', ext_options_extra(source_urls = ['http://rstan.org/repo/src/contrib/'])),
```

Comment:
@Bart-CER: an easyconfig for R v3.1.2 was just merged in with #1289

Can you sync up with that one? The commented out extensions included in here also need to be tackled/explained/removed.

I'll be happy to help out with this, the R extensions stuff can be ... daunting. ;-)